sway/commands/bar/pango_markup.c

```diff
@@ -9,10 +9,7 @@ struct cmd_results *bar_cmd_pango_markup(int argc, char **argv) {
 	if ((error = checkarg(argc, "pango_markup", EXPECTED_EQUAL_TO, 1))) {
 		return error;
 	}
-	if (!config->current_bar) {
-		return cmd_results_new(CMD_FAILURE, "No bar defined.");
-	}
-	config->current_bar->pango_markup 
+	config->current_bar->pango_markup
```

Comment:
nit: since the trailing space is being striped from this line anyway, can you move the `=` up to this line?